adam/curriculum/phase1_curriculum.py

```diff
@@ -1041,6 +1042,52 @@ def make_jump_template(
     )
 
 
+def make_pass_template(
+    agent: TemplateObjectVariable = standard_object(
+        "thrower_0", THING, required_properties=[ANIMATE]
+    ),
+    theme: TemplateObjectVariable = standard_object(
+        "object_0", required_properties=[INANIMATE]
+    ),
+    goal: TemplateObjectVariable = standard_object(
+        "catcher_0", THING, required_properties=[ANIMATE]
+    ),
```

Comment:
This was me not clearly communicating my intent. We'd prefer not to provide a default for these minimal requirements of the template. Rather the function that creates the curriculum should be responsible for creating these as providing them to the template. This needs to be adjusted on pass, take, and push